### PR TITLE
chore: upstream some minor fixes necessary for creating releases

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## WHAT

This pr adds some minor fixes and tweaks in preparation for releasing versions, such as adapting GH Actions workflows and adding some files needed for QGate evaluation.

## WHY

Those changes are necessary for releasing versions and must thus be upstreamed.

## IMPORTANT
~~This pull-request should be merged _after_ https://github.com/catenax-ng/tx-tractusx-edc/pull/10 is merged, because that contains some last-minute fixes as well.~~

~~[edit] This PR should also be merged _after_ #134 to avoid duplicated work~~ 

done

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,10 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ## [Unreleased]
 
+## [0.3.1] - 2023-03-27
+
 ### Added
 
 ### Changed
-- Support unauthenticated access to the ObservabilityAPI (#126)
+
+-   Support unauthenticated access to the ObservabilityAPI (#126)
```
Comment (florianrusch-zf) [inline CHANGELOG.md:16]:
```suggestion
- Support unauthenticated access to the ObservabilityAPI (#126)
```

Comment (florianrusch-zf) [reply CHANGELOG.md:16]: The same formatting error appears within the whole file. Can you fix them all?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ assignees: ''
 
 ---
 
-_If you are missing a feature or have an idea how to improve this project that should first be
+_If you are missing a feature or have an idea how to improve this project that should first be
```
Comment (florianrusch-zf) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
_If you are missing a feature or have an idea how to improve this project that should first be
```

File: misc/NOTICE.md.template [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Notices for Catena-X NG Product EDC
```
Comment (florianrusch-zf) [inline misc/NOTICE.md.template:1]:
```suggestion
# Notices for Tractus-X EDC
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

**Review by florianrusch-zf — APPROVED** [review]

File: charts/edc-controlplane/README.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ helm install my-release tractusx-edc/edc-controlplane --version 0.3.0
 | envSecretName | string | `nil` | [Kubernetes Secret Resource](https://kubernetes.io/docs/concepts/configuration/secret/) name to load environment variables from |
 | fullnameOverride | string | `""` | Overrides the releases full name |
 | image.pullPolicy | string | `"IfNotPresent"` | [Kubernetes image pull policy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) to use |
-| image.repository | string | `"ghcr.io/catenax-ng/product-edc/edc-controlplane-postgresql-hashicorp-vault"` | Which derivate of the edc control-plane to use. One of: [ghcr.io/catenax-ng/product-edc/edc-controlplane-postgresql-hashicorp-vault, ghcr.io/catenax-ng/product-edc/edc-controlplane-postgresql, ghcr.io/catenax-ng/product-edc/edc-controlplane-memory] |
+| image.repository | string | `"ghcr.io/eclipse-tractusx/tractusx-edc/edc-controlplane-postgresql-hashicorp-vault"` | Which derivate of the edc control-plane to use. One of: [ghcr.io/eclipse-tractusx/tractusx-edc/edc-controlplane-postgresql-hashicorp-vault, ghcr.io/eclipse-tractusx/tractusx-edc/edc-controlplane-postgresql, ghcr.io/eclipse-tractusx/tractusx-edc/edc-controlplane-memory] |
```
[the remark this answers is not in the document]
Comment (SebastianBezold) [reply charts/edc-controlplane/README.md:51]: Hi @florianrusch-zf and @paullatzelsperger ,
images is eclipse-tractusx are private on default, and I think they will stay that way. 
In the meantime we have the proper docs/notices/agreements, that we can publish images on DockerHub.
#149 was a precondition for that. So in my opinion you are good to go and publish on DockerHub, if you add the proper notice. Example: https://github.com/eclipse-tractusx/item-relationship-service#notice-for-docker-image

**Pull request merged**